scripts/install/install-linux.sh

```diff
@@ -14,14 +14,16 @@ chmod +x $PORTER_HOME/porter
 cp $PORTER_HOME/porter $PORTER_HOME/porter-runtime
 echo Installed `$PORTER_HOME/porter version`
 
-$PORTER_HOME/porter mixin install exec --version $MIXIN_PERMALINK
-$PORTER_HOME/porter mixin install kubernetes --version $MIXIN_PERMALINK
-$PORTER_HOME/porter mixin install helm --version $MIXIN_PERMALINK
-$PORTER_HOME/porter mixin install azure --version $MIXIN_PERMALINK
-$PORTER_HOME/porter mixin install terraform --version $MIXIN_PERMALINK
-$PORTER_HOME/porter mixin install az --version $MIXIN_PERMALINK
-$PORTER_HOME/porter mixin install aws --version $MIXIN_PERMALINK
-$PORTER_HOME/porter mixin install gcloud --version $MIXIN_PERMALINK
+$PORTER_HOME/porter mixin install exec --version $PKG_PERMALINK
+$PORTER_HOME/porter mixin install kubernetes --version $PKG_PERMALINK
+$PORTER_HOME/porter mixin install helm --version $PKG_PERMALINK
+$PORTER_HOME/porter mixin install azure --version $PKG_PERMALINK
```

Comment:
Should we change this to `arm`?  (Would apply to the other install scripts...)

Reply:
Thanks! Yes, that should have happened a while ago but no time like the present!